qa-tests-backend/src/main/groovy/services/NetworkGraphService.groovy

```diff
@@ -45,75 +43,50 @@ class NetworkGraphService extends BaseService {
         }
     }
 
+    @Retry
```

Comment:
What is the default number of retries and the pause duration between them?
Could we maybe specify it explicitly?

Reply:
What values would you like to see here? I think the defaults are ok (3 retries, 1s)
https://github.com/stackrox/stackrox/blob/dfeb3ed19d781e0649b2725a2a136307a5ba6624/qa-tests-backend/annotations/src/main/groovy/io/stackrox/annotations/Retry.groovy#L41-L51

Reply:
To be completely honest, the old way of retrying was much clearer to me. Waiting 3 times for 1 second feels to short for a test where changes need to propagate from Collector through Sensor, where the Sensor processes those changes only twice per minute based on a ticker.